pom.xml

```diff
@@ -194,17 +194,12 @@
           <logViolationsToConsole>true</logViolationsToConsole>
```

Comment:
I don't think this is related, but I'm not sure the custom overrides are required anymore. We could remove them, leaving just the suppressionsLocation